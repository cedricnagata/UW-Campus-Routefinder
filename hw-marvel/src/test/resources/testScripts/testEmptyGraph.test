# Test ParseData and findPath on empty file

LoadGraph emptyGraph empty.csv

ListNodes emptyGraph

FindPath emptyGraph node_a node_b