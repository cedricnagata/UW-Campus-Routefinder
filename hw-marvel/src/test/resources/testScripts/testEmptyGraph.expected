# Test ParseData and findPath on empty file

loaded graph emptyGraph

emptyGraph contains:

unknown: node a
unknown: node b